tests/integration/django/dill/leaves/features/steps.py

```diff
@@ -78,8 +77,8 @@ def count_harvesters(step):
     print("Harvester count: %d" % Harvester.objects.count())
 
 
-@creates_models(Panda)
```

Comment:
This was specifically here to test that `creates_models`, though deprecated, still works.